components/tag/stories/tag.stories.js

```diff
@@ -100,15 +103,24 @@ export default {
 		},
 		packageJson,
 		metadata,
+		downState: {
+			selectors: [".spectrum-Tag:not(:disabled)"],
+		},
+	},
+	decorators: [
+		withDownStateDimensionCapture,
+	],
+	status: {
+		type: "migrated",
 	},
```

Comment:
This `status` object should go in `parameters` (at line 109ish) in order to get picked up properly- sorry I didn't clarify in my last comment.

Reply:
I definitely should have been checking to see if this was doing anything 😂